apertium-tur-uzb.tur-uzb.dix

```diff
@@ -4326,6 +4337,8 @@
        <e><p><l>kaç<s n="adv"/></l><r>nechta<s n="adv"/></r></p></e>
        <e><p><l>neredeyse<s n="adv"/></l><r>qariyb<s n="adv"/></r></p></e>
        <e><p><l>daim<s n="adv"/></l><r>doim<s n="adv"/></r></p></e>
+       <e><p><l>her<b/>daim<s n="adv"/></l><r>har<b/>doim<s n="adv"/></r></p></e>
+       <e><p><l>her<b/>zaman<s n="adv"/></l><r>har<b/>doim<s n="adv"/></r></p></e>
```

Comment:
We might want all these in the appropriate lsx files too.

Reply:
"kaç" is not an adverb, it is a verb and DET-QNT

Reply:
Verb form already exists: 
   <e><p><l>kaç<s n="v"/><s n="tv"/></l><r>qoch<s n="v"/><s n="iv"/></r></p></e>
So, this one has been changed to det-qnt:
   <e><p><l>kaç<s n="det"/><s n="qnt"/></l><r>nechta<s n="det"/><s n="qnt"/></r></p></e>